### PR TITLE
nix: rm leftover dapp2 refs

### LINKED ISSUE #921: 'dapp2' missing, at /nix/store/.../nix/solidity-package.nix:23:19
Shouldn't this line be removed after `dapp2` was deleted:
https://github.com/dapphub/dapptools/blob/master/nix/solidity-package.nix#L23?

Commit: https://github.com/dapphub/dapptools/commit/850ce46528ec2c1425c0de63e24e86767471849b

### PR DESCRIPTION
**Pull request by d-xo**
Should fix #921 

Can you confirm @gbalabasquer ?

### CODE REVIEW

File: nix/solidity-package.sh [hunk 1]
```diff
@@ -17,8 +17,8 @@ mapfile -t files < <(find "$dir/src" -name '*.sol')
 json_file="out/dapp.sol.json"
 (set -x; solc $REMAPPINGS "${opts[@]}" $solcFlags "${files[@]}" > "$json_file")
 
-if [[ "$doCheck" == 1 ]] && command -v dapp2-test-hevm >/dev/null 2>&1; then
-  DAPP_OUT=out dapp2-test-hevm
+if [[ "$doCheck" == 1 ]] then
```
Comment (gbalabasquer) [inline nix/solidity-package.sh:20]: I'm getting:
```
line 20: syntax error near unexpected token `then'
```

I think this should fix it:

```suggestion
if [[ "$doCheck" == 1 ]]; then
```

Comment (gbalabasquer) [inline nix/solidity-package.sh:20]: I'm getting:
```
line 20: syntax error near unexpected token `then'
```

I think this should fix it:

```suggestion
if [[ "$doCheck" == 1 ]]; then
```

Comment (gbalabasquer) [reply nix/solidity-package.sh:20]: @d-xo ^^

Comment (d-xo) [reply nix/solidity-package.sh]: thanks!

**Pull request merged**